todo/src/components/TodoApp.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { addTodo, toggleCompleted, deleteTodo } from "../actions/index";
+
+import "./TodoApp.css";
+
+class TodoApp extends Component {
+  state = {
+    newTodo: ""
+  };
```

Comment:
good. You realize that we need local state to handle changes in form as opposed to going to the store. Great job!